whylog/tests/tests_log_reader/test_read.py

```diff
@@ -29,6 +32,19 @@ def setUpClass(cls):
             )
         )  # yapf: disable
 
+    def test_getting_line_by_offset(self):
```

Comment:
I've improved function readability. Please check it again and tell if another test is really necessary.